### PR TITLE
Notebooks: Add Command + Keyboard Shortcut to Clear Outputs of Active Cell

### PR DESCRIPTION
**Pull request by chlafreniere**
Implements #6019.

Chose Ctrl+Shift+R as Ctrl+Shift+O was already taken. In Jupyter, they use Ctrl+O, which obviously won't work.

Added a command to the command palette, and gave it a keybinding. Also added an integration test.

### CODE REVIEW

File: extensions/integration-tests/src/notebook.test.ts [hunk 1]
```diff
@@ -380,6 +389,18 @@ class NotebookTester {
 		assert(clearedOutputs, 'Outputs of all the code cells from Python notebook should be cleared');
 		console.log('After clearing cell outputs');
 	}
+
+	async verifyClearOutputs(notebook: azdata.nb.NotebookEditor): Promise<void> {
+		let cellWithOutputs = notebook.document.cells[0].contents && notebook.document.cells[0].contents.outputs && notebook.document.cells[0].contents.outputs.length > 0;
+		assert(cellWithOutputs === true, 'First notebook cell has no outputs');
```
Comment (kevcunnane) [inline extensions/integration-tests/src/notebook.test.ts]: Minor: usually expect messages to be in form "expected cell to have outputs", not "this has no outputs"

File: extensions/integration-tests/src/notebook.test.ts [hunk 2]
```diff
@@ -380,6 +389,18 @@ class NotebookTester {
...
+		console.log('Before clearing cell outputs');
+		let clearedOutputs = await notebook.clearOutput(notebook.document.cells[0]);
+		let firstCell = notebook.document.cells[0];
+		assert(firstCell.contents && firstCell.contents.outputs && firstCell.contents.outputs.length === 0, `Expected Output: 0, Actual: '${firstCell.contents.outputs.length}'`);
+		assert(clearedOutputs, 'Outputs of requested code cell should be cleared');
+		console.log('After clearing cell outputs');
```
Comment (kevcunnane) [inline extensions/integration-tests/src/notebook.test.ts]: Do we really need these? They fill up the cmdline when running other tests. Can we get rid of the console.log statements (or comment out if you want to re-enable later)?

File: src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts [hunk 3]
```diff
@@ -384,6 +391,28 @@ export class MainThreadNotebookDocumentsAndEditors extends Disposable implements
...
+		if (cellUri) {
+			let uriString = URI.revive(cellUri).toString();
+			cell = editor.cells.find(c => c.cellUri.toString() === uriString);
+			// If it's markdown what should we do? Show notification??
+		} else {
+			// Use the active cell in this case, or 1st cell if there's none active
+			cell = editor.model.activeCell;
+		}
+		if (!cell || (cell && cell.cellType !== CellTypes.Code)) {
+			return Promise.reject(new Error(localize('runActiveCell', "F5 shortcut key requires a code cell to be selected. Please select a code cell to run.")));
```
Comment (kevcunnane) [inline src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts]: Error message is for runActiveCell

Comment (kevcunnane) [reply src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts]: Also probably just needs to be a warning / info

File: extensions/integration-tests/src/notebook.test.ts (hunk 1 above)
Comment (chlafreniere) [reply extensions/integration-tests/src/notebook.test.ts]: Thanks

File: src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts (hunk 3 above)
Comment (chlafreniere) [reply src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts]: Good catch

File: extensions/integration-tests/src/notebook.test.ts (hunk 2 above)
Comment (chlafreniere) [reply extensions/integration-tests/src/notebook.test.ts]: Agreed, don't need them anymore.

**Pull request merged**